### ISSUE TITLE
How to record livestream mp4

### ISSUE THREAD

**Issue by eliich** [zerodytrash/TikTok-Live-Connector#159]
How can I record the livestream mp4? Is that possible in this project?

**Comment by ceoofdysfunction** [comment]
Yes. You can use the project to get the .FLV url then something like ffmpeg to save it in whatever format you would like.

**Comment by eliich** [comment]
> Yes. You can use the project to get the .FLV url then something like ffmpeg to save it in whatever format you would like.

Can you explain?

**Comment by ymerkos** [comment]
on the base html page there is a json that has a link to the stream. its an FLV so u just need to continously request that url to get the flv then convert it afterwards
